components/eam/docs/user-guide/index.md

```diff
@@ -1 +1,175 @@
+change the *compset* and *grid*.
+See the
+[Case Control System Basic Usage](https://esmci.github.io/cime/versions/master/html/users_guide/index.html#case-control-system-part-1-basic-usage) for general descriptions of compsets and grids.
+
+Certain namelist paramaters, input data files, and output file specifcations can also be modified.
+These are described below as ways to customize runs.
+
+Step-by-step instructions on how to run and analyze E3SM with a script can be found at
+[E3SM step-by-step guide](https://docs.e3sm.org/running-e3sm-guide/)
+
+## Scientifically supported compsets and grids
+
+### Compsets
+
+All of the compsets below run with the complete set of E3SM atmospheric configuration of EAMV3. For more information on the schemes in EAMv3, see the [Technical Guide](../tech-guide/index.md)
+
+`F2010` - Climatological present day climate (year 2010)
+
+`F1850` - Climatological pre-industrial-day climate (year 1850)
+
+`F20TR` - Historical EAM simulation with time varying sea-surface temperatures, aerosol emissions, and greenhouse gas forcings (year 1850-2014)
+
+### Grids
+
+Only one grid combination is currently supported for the above compsets:
+
+`ne30pg2_r05_IcoswISC30E3r5` - ne30pg2 atmosphere, 0.5deg x 0.5deg land grid, and Icosahedral 30 km mesh with ice shelves cavities (wISC), E3SMv3 (E3) revision r5
+
+## Customizing runs
+
+### Compile-time options
+
+Some customizations require making changes before the model is built.
+
+### Run-time options
+
+Run-time customization is enabled by a Fortran namelist.
+
+Namelist parameters can be changed from default values by putting them in the `user_nl_eam` file in the case directory
+with the desired new value.
+
+This [Table of Namelist Parameters](namelist_parameters.md) includes many of the paramaters that control
+physics schemes described in the [tech-guide](../tech-guide/index.md)
+
+#### History File Namelist Parameters
+
+By default, EAM will output a set of monthly-averaged variables. Additional output files can be specified using the following flags in the `user_nl_eam` file:
+
+`finclX` - List of variables (in single quotes and separated by commas) that are added to tape X.
+
+`fexclX` - List of variables (in single quotes and separated by commas) that will be excluded in tape X.
+
+`nhtfrq` - List of write frequencies for each of the history files. A value of 0 denotes a monthly frequency. Negative values denote hourly frequencies (e.g., `-3` will write an output every 3 hours). Positive values denotes the frequency in model timesteps (e.g., `4` will write an output every 4 timesteps).
+
+`mfilt` - List that sets the number of timesteps to write in a single file before starting a new file.
+
+`avgflag_pertape` - List that sets the type of output to write. Choices are `'A'` for time-averaged output, `'A'` for instantaneous output, `'MIN'` for time-minimum output, and `'MAX'` for time-maximum output.
```

Comment:
Need to replace `MIN` with `M`, and `MAX` with `X` for the avgflag above.

Reply:
Done.